examples/warm_start.py

```diff
@@ -0,0 +1,32 @@
+"""
+An example training a LogisticRegression model, performing grid search
+using TuneGridSearchCV.
+
+This example uses early stopping to further improve runtimes
+by eliminating worse hyperparameter choices early based off
+of its average test score from cross validation.
+"""
+
+from tune_sklearn import TuneGridSearchCV
+from sklearn.model_selection import train_test_split
+from sklearn.datasets import load_iris
+from sklearn.linear_model import LogisticRegression
+import numpy as np
+
+x, y = load_iris(return_X_y=True)
+x_train, x_test, y_train, y_test = train_test_split(x, y, test_size=.2)
+
+clf = LogisticRegression()
+parameter_grid = {"C": [1e-4, 1e-1, 1, 2]}
+
+tune_search = TuneGridSearchCV(
+    clf,
+    parameter_grid,
+    early_stopping="MedianStoppingRule",
```

Comment:
can you make this early_stopping=True?

Median..rule is not a concept that most folks know about.